pom.xml

```diff
@@ -171,7 +171,7 @@
         <!--
          If you update a dependendy version so it is no longer a SNAPSHOT
```

Comment:
You missed this comment here right next to the version change.  If we want the shim for 3.0.1 to be released you need to change the snapshot-shims profile too. @tgravescs do you want handle this with your docs update, or should I handle it separately?